test/unit/index.spec.js

```diff
@@ -199,4 +199,13 @@ describe('api', () => {
       expect(api.options({})).toEqual({})
     })
   })
+
+  describe('help()', () => {
+    it('help', () => {
```

Comment:
Would be great to give more descriptive character of this test, like: `it throws an error if first argument is not a function`